src/EditorFeatures/VisualBasicTest/QuickInfo/SemanticQuickInfoSourceTests.vb

```diff
@@ -2032,5 +2032,32 @@ End Interface
             TestFromXml(markup, MainDescription("Interface IFoo"), Documentation("summary for interface IFoo"))
         End Sub
 
+        <Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)>
+        <WorkItem(4868, "https://github.com/dotnet/roslyn/issues/4868")>
+        Public Sub QuickInfoExceptions()
+            Test("
+Imports System
+Namespace MyNs
+    Class MyException1
+        Inherits Exception
+    End Class
+    Class MyException2
+        Inherits Exception
+    End Class
+    Class TestClass
+        ''' <exception cref=""MyException1""></exception>
+        ''' <exception cref=""T:MyNs.MyException2""></exception>
+        ''' <exception cref=""System.Int32""></exception>
+        ''' <exception cref=""Double""></exception>
+        ''' <exception cref=""Not_A_Class_And_Therefore_Not_Listed""></exception>
+        Sub M()
+            M$$()
+        End Sub
+    End Class
+End Namespace
+",
+                Exceptions($"{vbCrLf}{WorkspacesResources.Exceptions}{vbCrLf}  MyException1{vbCrLf}  MyException2{vbCrLf}  Double{vbCrLf}  Integer"))
```

Comment:
Unfortunately this isn't easily done.  The final display is all done via a `StackPanel` in `QuickInfoDisplayPanel.cs`, so the best we could do is by using the `.ToString()` overload which happens to generate the text in the same order as the `StackPanel`, but there's no guarantee that these two things (the order or the `StackPanel` and the order of evaluation in `.ToString()`) will stay in sync.

To make it even more complicated, `.ToString()` eventually evaluates the `System.Windows.Documents.Run::Text` property which inserts left-to-right Unicode markers, and while we could explicitly add those in the test, as soon as we run our tests on an RTL language then we're broken again.

For what it's worth, I did some manual testing with exceptions and await (which generates the `Usage:` section) and the display is consistent with VS2013.

**tl;dr;**
Not worth it.